examples/s3-import/variables.tf

```diff
@@ -1 +1,45 @@
 #(empty)
+variable "supporting_resources_name" {
+  type        = string
+  description = "The stack name for supporting resources launched separately"
+  default     = "terraform-aws-rds"
+}
+
+variable "region_1" {
```

Comment:
could we change the name of these two? `region_1` and `region_2`? asking because they cold be called something easier to reference, like `region` and `replication_bucket_region`.

Reply:
Addressed @mariomerco